Wordle.Api/Wordle.Api/Services/LeaderboardService.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.Reflection.Metadata.Ecma335;
+using Wordle.Api.Models;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace Wordle.Api.Services;
+
+public class LeaderboardService
+{
+    //private readonly List<string> words = LoadWordList();
+    private static object _lock = new();
+
+    public WordleDbContext Db { get; set; }
+
+    public LeaderboardService(WordleDbContext db)
+    {
+        Db = db;
+    }
+
+    public async Task<List<Score>> GetTopTenScores()
+    {
+        /*for (int j = 0; j < players.Count; j++)
```

Comment:
Since all this is commented out and you switched to a list, probably okay to delete?

Reply:
It is now, so I suppose so

Reply:
By the way, I have no clue why git decided to split the API into two folders - there was only one on my local repo. I've copied the whole thing over, deleted the extra folder on github, then moved everything back and that seems to have fixed it.